imports/plugins/included/payments-stripe/server/methods/stripe.js

```diff
@@ -124,7 +124,7 @@ Meteor.methods({
           response: chargeResult
         };
       } else {
-        Logger.info("Stripe Call succeeded but charge failed");
+        Logger.debug("Stripe Call succeeded but charge failed");
```

Comment:
No, this is just a normal charge failure and the error is passed back up the response chain.